test/fixtures/todo/json5/serialization/object.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'object with numeric literal as key',
+    value: { 42: true },
+    serialized: '{42:true}'
+  },
+  {
+    name: 'object with unquoted keys',
+    value: { $: 'dollar', _$_: 'multiple symbols' },
+    serialized: '{$:\'dollar\',_$_:\'multiple symbols\'}'
+  },
+  {
+    name: 'object with unicode unquoted key',
```

Comment:
Also, it is better to name this test case "object with identifier name, containing non-latin Unicode literals, as a key".